src/index.ts

```diff
@@ -97,13 +97,16 @@ const getOptionalInput = (name: string) => core.getInput(name) || undefined;
       return;
     }
     case hasChangesets:
-      await runVersion({
+      const { pullRequestNumber } = await runVersion({
         script: getOptionalInput("version"),
         githubToken,
         prTitle: getOptionalInput("title"),
         commitMessage: getOptionalInput("commit"),
         hasPublishScript,
       });
+
+      core.setOutput("pullRequestNumber", String(pullRequestNumber));
```

Comment:
Since this will always be int - could we not cast it with `String()`? Would it have any observable difference if this wouldn't be casted? Like perhaps one version would not be quoted or smth? Did you test this from the consumer's side of this action?

I've checked the source code of `@actions/core` and I think this gets stringified anyway but it would be great if we could confirm if there is any practical difference between both flavours - if there is not then I'm ok with leaving it as is.